common/src/main/scala/org/apache/celeborn/common/util/Utils.scala

```diff
@@ -476,14 +476,14 @@ object Utils extends Logging {
     hostPortParseResults.get(hostPort)
   }
 
-  private val MAX_DEFAULT_NETTY_THREADS = 64
+  private var maxDefaultNettyThreads = 64
```

Comment:
Do we want to simply eliminate this variable ?
Pass `conf` to `defaultNumThreads` and fetch from there ?